### ISSUE TITLE
LOGGER FAKE

### ISSUE THREAD

**Issue by twistyohnah** [prince01A/Pirate#1]
.exe LOL

**Comment by cfm-miku-en** [comment]
i agree, you cant read exe code, you have to use a tool to. if there is a .exe and a ,py version, i would choose the .py edition.
